test/integration/platforms/test_run_slurm_check_db.py

```diff
@@ -252,7 +262,7 @@ def _check_files_recovered(as_conf, log_dir, expected_files) -> dict:
     for f in log_dir.glob('*'):
         files_check_list[f.name] = not any(
             str(f).endswith(f".{i}.err") or str(f).endswith(f".{i}.out") for i in range(retrials + 1))
-    stat_files = [str(f).split("_")[-1] for f in log_dir.glob('*') if "STAT" in str(f)]
+    stat_files = [str(f).split("_")[-1] for f in log_dir.parent.glob('*') if "STAT" in str(f)]
```

Comment:
Took me a long time to find this :laughing: 

The previous test was made for the results of running wrappers with the local platform. So it was working as the STAT_0 file existed in the log_dir. However, with a slurm platform, the log_dir in the remote contains the file, and then the log recovery will download it to tmp, not to the tmp/log_<EXPID> :face_exhaling: